testsuite/sequential_execution_performance.py

```diff
@@ -4,13 +4,13 @@
 
 # Set the tps threshold for block size 1k, 10k and 50k
 BLOCK_SIZES = ["1k", "10k", "50k"]
```

Comment:
should be pass block sizes in, instead of needing to rely on them being the same here and the defaults in the code?

Reply:
Not sure If I understand, can you clarify?

Reply:
I mean this relies on this line above matching and not being changed:
  let block_sizes = opt.block_sizes.unwrap_or_else(|| vec![1000, 10000, 50000]);
  
can we just pass in the --block-sizes param to the command here?

Reply:
Yes, good point. Let me sneak that change in the follow-up change I am working on - that will change this file anyways.